src/util.ts

```diff
@@ -225,3 +225,22 @@ let globalTSPattern = /(^|\/)globals?\.d\.ts$/;
 export function isGlobalTSFile(filename: string): boolean {
 	return globalTSPattern.test(filename);
 }
+
+
+/*
+ * WorkspaceReferenceParams holds parameters for the
```

Comment:
I'd move them to `request-type.ts`, where similar `WorkspaceSymbolParamsWithLimit` is